mmv1/third_party/terraform/services/vmwareengine/resource_vmwareengine_subnet_test.go

```diff
@@ -8,6 +8,8 @@ import (
 )
 
 func TestAccVmwareengineSubnet_vmwareEngineUserDefinedSubnetUpdate(t *testing.T) {
+	// Temporarily skipping so that this test does not run and consume resources during PR pushes. It is bound to fail and is being fixed by PR #10992
```

Comment:
we will remove these before submitting right?

Reply:
We can keep these commented till PR #10992 gets submitted. That way, the consumption of nodes for tests which are bound to fail would be prevented. 

These tests would be modified and combined by PR #10992 which is built on the design [go/gcve-resource-optimisation-for-tf-tests](http://goto.google.com/gcve-resource-optimisation-for-tf-tests)

Reply:
Understood, o the relevant test for these code changes are all in `mmv1/third_party/terraform/services/vmwareengine/resource_vmwareengine_private_cloud_test.go`

Reply:
Yes, thats correct